### ISSUE TITLE
Gradle build lacks Testcontainers dependency

### ISSUE THREAD

**Issue by timtebeek** [spring-projects/spring-petclinic#1267]
Following last week's addition of testcontainers https://github.com/spring-projects/spring-petclinic/commit/3c1ef72e6d45ef41fbc86f51476e5a72e9e66564 the Gradle build now fails with compilation errors due to a missing dependency there.
```
> Task :compileTestJava FAILED
/tmp/spring-petclinic/src/test/java/org/springframework/samples/petclinic/MySqlIntegrationTests.java:26: error: package org.springframework.boot.testcontainers.service.connection does not exist
import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
                                                                 ^
/tmp/spring-petclinic/src/test/java/org/springframework/samples/petclinic/MySqlIntegrationTests.java:34: error: package org.testcontainers.containers does not exist
import org.testcontainers.containers.MySQLContainer;
                                    ^
...
```

It's likely that the same dependencies need to be added to `build.gradle`, such that it compiles and runs again.

I'd like to propose to also [add a Gradle build to the GitHub Actions workflows](https://github.com/spring-projects/spring-petclinic/tree/main/.github/workflows), such that it's tested in parallel.

**Comment by dsyer** [comment]
Good idea. Should be trivial if anyone wants to send a pull request.

**Comment by dsyer** [comment]
Fixed in f85ad88
